maths/palindromic_number_checker.py

```diff
@@ -0,0 +1,24 @@
+def is_number_palindromic(num: int) -> bool:
+    """
+    This function checks if given number is palindromic or not.
+    >>> is_number_palindromic(121)
+    True
+    >>> is_number_palindromic(1231)
+    False
+    >>> is_number_palindromic(12321)
+    True
+    >>> is_number_palindromic(1211)
+    False
+    """
+    num_rev_str = str(num)[::-1]
+    num_rev = int(num_rev_str)
+    if num == num_rev:
```

Comment:
might be worth it just to replace the variable with the value since it's just used in one place. But that's just my preference this looks more readable tho